build.gradle

```diff
@@ -16,7 +38,12 @@ apply plugin: 'java'
 apply plugin: 'checkstyle'
 apply plugin: 'jacoco'
 apply plugin: 'ru.vyarus.animalsniffer'
-apply plugin: 'nebula.rxjava-project'
+apply plugin: 'maven'
+apply plugin: 'osgi'
+apply plugin: "me.champeau.gradle.jmh"
```

Comment:
super nit: can you please use either single or double quotes for consistency? 

(Separate PR was always overkill for that, but since you're modifying the files it feels like a great opportunity to finally do that)